Makefile

```diff
@@ -2,7 +2,9 @@
 
 # Using most recent spack version with preferred version of octopus
 spack-develop:
-	docker build -f Dockerfile --build-arg SPACK_VERSION=develop -t octopus-spack  .
+	# Setting BERKELEYGW_VER to 3.0.1 as v4 is released but the download link is broken!
```

Comment:
I thought Octopus would prefer not to use v4 (but rather 3.10 - not sure about the precise number). Can you pdate the comment and version accordingly?

Reply:
Actually, version 4 was released just this month and Octopus developers themselves haven't tested this https://gitlab.com/octopus-code/octopus/-/issues/1003 I am reluctant to make changes to the `package.py` or make a comment here until the octopus developers themself confirm the support or absence of it. You are right that the previous recommended version here is v3.0.1 as is written in the comment.